spring-cloud-gcp-data-firestore/src/main/java/org/springframework/cloud/gcp/data/firestore/FirestoreTemplate.java

```diff
@@ -189,6 +193,26 @@ public boolean isUsingStreamTokens() {
 		return saveAll(Mono.just(entity)).next();
 	}
 
+	private <T> Flux<T> create(Publisher<T> entities) {
+		return Flux.from(entities).flatMap(this::create);
+	}
+
+	private <T> Mono<T> create(T entity) {
```

Comment:
Would be nice to add a javadoc for this one describing purpose of method something like "Creates a single (non-batched) document in Firestore; ID generation is only available for non-batched created documents."